.github/workflows/regression.yml

```diff
@@ -45,7 +51,7 @@ jobs:
           key: ${{ matrix.config_set }}-${{ github.sha }}
           restore-keys: ${{ matrix.config_set }}
       - name: Build 
-        uses: docker://su2code/build-su2:20191105
+        uses: docker://ghcr.io/martin-g/su2code/build-su2:2
```

Comment:
Related to https://github.com/su2code/Docker-Builds/pull/1. I need to use the Docker image on the self-hosted Linux ARM64 runner and this requires updating the base image to Ubuntu 20.04 that is multi-platform.

The url to the Docker image will be updated to an official one once https://github.com/su2code/Docker-Builds/pull/1 is merged.